package.json

```diff
@@ -27,6 +27,7 @@
   "dependencies": {
     "@iconify/iconify": "^2.0.0-rc.1",
     "@vueuse/core": "^4.0.0-beta.15",
+    "copy-text-to-clipboard": "afzalsayed96/copy-text-to-clipboard",
```

Comment:
any reason for this to be a git repo instead of an npm package?

Reply:
https://github.com/antfu/icones/pull/19#issuecomment-703097341